app/js/services/situationService.js

```diff
@@ -120,6 +120,7 @@ angular.module('ddsCommon').factory('SituationService', function($http, $session
             }
 
             var result = {
+                referenceId: situation._id,
```

Comment:
I used that specific word to reflect OpenFisca naming convention. I agree with your point but given OpenFisca's definition of reference, it makes sense to be consistent.